### PR TITLE
Change the access level of variables in BoxViewController

### PR DESCRIPTION
**Pull request by gtsiap**
These variables can be useful if we need to subclass `BoxViewController`.

### CODE REVIEW

**Review by TheAdamBorek — APPROVED** [review]
It's still `let` so I'm fine with that. I think @andersio can also look at this

File: BentoKit/BentoKit/Screen/BoxViewController.swift [hunk 1]
```diff
@@ -7,8 +7,8 @@ import UIKit
 open class BoxViewController<ViewModel: BoxViewModel, Renderer: BoxRenderer, Appearance>
 : UIViewController where ViewModel.State == Renderer.State, ViewModel.Action == Renderer.Action, Renderer.Appearance == Appearance {
     public let viewModel: ViewModel
-    private let rendererConfig: Renderer.Config
-    private let appearance: Property<Appearance>
+    public let rendererConfig: Renderer.Config
+    public let appearance: Property<Appearance>
```
Comment (sergdort) [inline BentoKit/BentoKit/Screen/BoxViewController.swift:11]: Why do we need it to be `public`?

**Review by diegopetrucci — CHANGES_REQUESTED** [review]
Do we already have a need for this, or is it speculative?

**Review by diegopetrucci — APPROVED** [review]
Okay, just saw the referenced PR.

**Pull request merged**